packages/contracts/contracts/engine/V3/EngineFactoryV0.sol

```diff
@@ -33,6 +33,9 @@ contract EngineFactoryV0 is Ownable, IEngineFactoryV0 {
     // public type
     bytes32 public constant type_ = "EngineFactoryV0";
 
+    // base URI prefix for all Engine contracts on this network/chainID
+    string public defaultBaseURIPrefix;
```

Comment:
nit: should we call this `defaultBaseURIHost`?

Reply:
YES! updated in [3040145](https://github.com/ArtBlocks/artblocks-contracts/pull/1549/commits/3040145ef1841d6e514e9657c195d62234715905)